scripts/solve_network.py

```diff
@@ -986,6 +990,19 @@ def extra_functionality(n, snapshots):
         custom_extra_functionality(n, snapshots, snakemake)
 
 
+def check_objective_value(n, solving):
+    check_objective = solving["check_objective"]
+    if check_objective["enable"]:
+        atol = check_objective["atol"]
+        rtol = check_objective["rtol"]
+        expected_value = check_objective["expected_value"]
+        if not np.isclose(n.objective, expected_value, atol=atol, rtol=rtol):
```

Comment:
Should we check `n.objective` or `n.statistics.capex().sum() + n.statistics.opex().sum()`?

Reply:
I would say objective just to have one loop less.